File: MediafyDemoAppJava/app/src/main/java/com/mediafy/demo/java/utils/ImageUtils.java


/*
 * Mediafy SDK
 *
 * Copyright 2024 Mars Technologies Ltd. All rights reserved.
 */

package com.mediafy.demo.java.utils;

import android.widget.ImageView;
import com.bumptech.glide.Glide;

public class ImageUtils {

    public static void download(String url, ImageView imageView) {
        Glide.with(imageView)
            .load(url)
            .into(imageView);
    }

}

File: MediafyDemoAppKotlin/app/src/main/java/com/mediafy/demo/kotlin/utils/ImageUtils.kt
/*
 * Mediafy SDK
 *
 * Copyright 2024 Mars Technologies Ltd. All rights reserved.
 */



package com.mediafy.demo.kotlin.utils

import android.widget.ImageView
import com.bumptech.glide.Glide

object ImageUtils {

    fun download(url: String, imageView: ImageView) {
        Glide.with(imageView)
            .load(url)
            .into(imageView)
    }

}